app/views/sessions/new.mobile.haml

```diff
@@ -27,7 +27,7 @@
             .control-group
               = f.label :username, t('username')
               .controls
-                = f.text_field :username, :autofocus => true
+                = f.text_field :username, :autofocus => true, :autocapitalize => 'none', :autocorrect => 'off'
```

Comment:
Please use ruby 1.9 hash syntax:
```ruby
= f.text_field :username, autofocus: true, autocapitalize: 'none', autocorrect: 'off'
```
(we [recenlty decided](https://www.loomio.org/d/bLH78pEh/styleguides) to use a [ruby styleguide](https://github.com/diaspora/diaspora/blob/develop/.rubocop.yml))